src/macos/Input.ts

```diff
@@ -7,7 +7,7 @@
 * MacOS implementation of Input interface.
 */
 
-import RN = require('react-native');
+//import RN = require('react-native');
```

Comment:
Why not remove?

Reply:
When they build this component, they'll want this line back.